pluto/src/vis/schematic/registry.ts

```diff
@@ -1020,18 +1633,34 @@ export const SYMBOLS: Record<Variant, Spec<any>> = {
   threeWayValve,
   fourWayValve,
   angledValve,
+  ballValve,
+  threeWayBallValve,
+  gateValve,
+  butterflyValveOne,
+  butterflyValveTwo,
+  breatherValve,
   manualValve,
   needleValve,
   reliefValve,
   angledReliefValve,
   checkValve,
   regulator,
   electricRegulator,
+  springLoadedReliefValve,
+  angledSpringLoadedReliefValve,
   pump,
   pistonPump,
   screwPump,
   cavityPump,
-  vacuumPump,
+  diaphragmPump,
+  ejectionPump,
+  // vacuumPump,
```

Comment:
uncomment

Reply:
Vacuum Pump was commented out because it was replaced by Compressor, could be changed back but we should remove Compressor so there aren't duplicates under different names.

Reply:
We need to keep vacuum pump because we have customers using it, could you instead remove the compressor? If you want, you can change the name and default label of the vacuum pump to be compressor, but the key in `registry.ts` and the `aetherKey` in `Symbols.tsx` needs to remain the same as the existing vacuum pump key.